### PR TITLE
move resource restart to pcs.lib

### PR DESCRIPTION
**Pull request by tomjelinek**

### CODE REVIEW

File: pcs_test/tier0/lib/commands/resource/test_restart.py
Comment (mirecheck) [inline pcs_test/tier0/lib/commands/resource/test_restart.py:1]: this added file is missing in 'pcs_test/Makefile.am'

**Review by mirecheck — CHANGES_REQUESTED** [review]

**Comment by tomjelinek** [comment]
retest this please

**Comment by tomjelinek** [comment]
retest this please

**Comment by tomjelinek** [comment]
retest this please

**Pull request merged**